moveit_ros/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -99,7 +99,7 @@ void OccupancyMapMonitor::initialize()
                                                      << "\" specified but no TF instance (buffer) specified. "
                                                         "No transforms will be applied to received data.");
 
-  tree_.reset(new OccMapTree(map_resolution_));
+  tree_.reset(new collision_detection::OccMapTree(map_resolution_));
```

Comment:
please use make_shared instead of reset(new

```suggestion
  tree_ = std::make_shared<collision_detection::OccMapTree>(map_resolution_);
```